src/App.vue

```diff
@@ -9,7 +9,7 @@
                     <unit v-for="unit in units" :unit="unit" :tileSize="gameBoard.tileSize" :key="unit.color"></unit>
                 </div>
                 <div style="position: absolute; bottom: 0">
-                    <timer v-if="game.state === 'prize' || game.state === 'exit'" :game-duration="game.timer"/>
+                    <timer v-if="game.state === 'prize' || game.state === 'exit' || game.state === 'time'" :game-duration="game.timer"/>
```

Comment:
Might be a good time to move this logic into a computed property with a nice semantic name like `gameInProgress`?
```
v-if="gameInProgress"
```

Now that I think about it, if we move that logic into the store as a getter, we can use it in `UnitControls` and remove that duplication as well.

Reply:
yeah i agree... i think i'll have to refactor some code here with the states to prevent the impending spaghetti code